guides/v2.3/cloud/project/project-upgrade.md

```diff
@@ -57,7 +57,10 @@ If you use PHP version 7.2, you must remove the `mcrypt` extension from the [`ex
 
 ## Verify and upgrade your extensions {#extensions}
 
-If you need to upgrade any third-party extensions and modules that support version 2.3, we recommend working in a new Integration branch with your extensions disabled. Review your third-party extension and module pages in Marketplace or other company sites to verify support for {{site.data.var.ee}} 2.3.
+{: .bs-callout-info }
+Review your third-party extension and module pages in Marketplace or other company sites to verify support for {{site.data.var.ee}} 2.3.
```

Comment:
Multi-line notes require different formatting to appear properly:

```
{%
include note.html
type='info'
content='Review your third-party extension and module pages in Marketplace or other company sites to verify support for {{site.data.var.ee}} and {{site.data.var.ece}} version 2.2.
  
If you need to upgrade any third-party extensions and modules that support version 2.2, we recommend working in a new Integration branch with your extensions disabled.'
%}```

Reply:
**implemented**

Thank you , @jeff-matthews